src/pages/components/cta-section.mdx

```diff
@@ -23,9 +24,21 @@ CTA (call-to-action) section is divided into two sub-sections. The first consist
 
 ![default](../..//images/pattern/cta-section/cta-section.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) | Notes |
+| -------- | --------- | ---------- | ---------- | ----------| ----------|
+| Headline | Short copy | Yes | 1 | XXS (40 / 55)| - |
+| Paragraph | Long copy | No | 1 | L (150 / 200)| - |
+| [Button group](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/button-group) | Component | Yes | 1-2 buttons | - | - |
```

Comment:
Is it worth calling out content block here for these like you're doing with link list and content items?

Reply:
I think this goes back to our previous conversation. I don't think it would necessarily be helpful to a user to see content block here, but up for thoughts.